hull3/marker_functions.sqf

```diff
@@ -34,7 +34,7 @@ hull3_marker_fnc_disableFireTeamMarkers = {
 };
 
 hull3_marker_fnc_initMarker = {
-    FUN_ARGS_3(_unit,_markerText,_markerColor);
+    params ["_unit","_markerText","_markerColor"];
```

Comment:
Please add space between commas. The original didn't have spaces, because of it was a macro. Same goes for all other arguments.